integration_test/README.md

```diff
@@ -137,7 +135,7 @@ The test is designed to be highly parameterizable. It reads various files from
 `third_party_apps_data` and decides what to do based on their contents. First
 it reads `test_config.yaml` and uses that to set some testing options. See the
 "test_config.yaml" section below. Then it reads
-`agent/ops-agent/<platform>/supported_applications.txt` to determine
```

Comment:
this change should have happened in a previous PR: https://github.com/GoogleCloudPlatform/ops-agent/pull/484.

I decided to include it here as I was going over the docs.